src/component/books/Showbooks.js

```diff
@@ -1,8 +1,14 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable react/prop-types */
 /* eslint-disable react/destructuring-assignment */
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { remove } from '../../redux/books/configBooks';
 
 const Showbooks = (props) => {
+  const dispatch = useDispatch();
+  const removeBookAction = bindActionCreators(remove, dispatch);
   const homeStyles = [
     {
       display: 'flex',
```

Comment:
- [ ] Do not use inline styles. Keep your style definition in a separate CSS file.

Reply:
- [ ] You have more inline styles in showBooks, please remowe them as well

Reply:
hey Asani, change this really necessary? I'm using inline style just because I've never used styles that way, if you notice I'm only using it in those documents, the others have styles in separate documents.

Reply:
> hey Asani, change this is really necessary? I'm using inline style just because I've never used styles that way, if you notice I'm only using it in those documents, the others have styles in separate documents.

Hei @xJokeer, it is listed under the general requirements that you cannot use inline styling   [general requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)